### ISSUE TITLE
Translation of tablet menus and tasks

### ISSUE THREAD

**Issue by AngusDLX** [ALiVEOS/ALiVE.OS#538]
After a month I succeeded in translating the mod into the Portuguese language, all fully translated modules. The "$STR Yes/No" missing in cfgvehicles.hpp are now also complete to facilitate translations into other languages in the future. The problem is that I was not able to translate the tablet's tasks and menus. (I'm familiar with stringtable.xml)

What is needed to create the translation in this case? I tried to observe the code but i not understand the information.

If anyone can explain it to me, I will do it with greater pleasure.

**Comment by xetra11** [comment]
Ask one of the devs to invite you to there slack channel. I think it is more appropiate to ask such stuff since ur are helping out in a technical way

**Comment by marceldev89** [comment]
I'm guessing that a good chunk of the strings in the GUIs are hardcoded and not going through a stringtable.xml. So step 1 would be to move them all to a stringtable.xml which just needs someone to do the actual work. ^^

If you want an invite to our slack then send someone on the team a PM on the forum and/or discord with your email and it'll be forwarded to the right person. :slightly_smiling_face:

**Comment by AngusDLX** [comment]
I would be very happy if I could! Nowadays, by looking at the files in more detail and calmly, I was able to transfer the strings from the tablet and the tasks to the stringtable.xml by looking at the Tupolov code.

I simply added localize "STR_BLAH_BLAH_BLAH" and worked perfectly( of course following ALiVE padronization), I tested a task, actions on the tablet and everything came out functional and translated including radio messages.

Is there a problem in this structure like Tupolov did with mil_ATO? Or would I have to carry all this on cfgVehicles?

**Comment by the-star** [comment]
@AngusDLX you're putting the translation into stringtable.xml for that specific folder? If so, I think you're on the right track.

Could you upload your stuff to your ALIVE fork? That way, if you get stuck you can link to the problem areas.

**Comment by AngusDLX** [comment]
> @AngusDLX you're putting the translation into stringtable.xml for that specific folder? If so, I think you're on the right track.
> 
> Could you upload your stuff to your ALIVE fork? That way, if you get stuck you can link to the problem areas.

Yes exactly in the specific folder, I will do mate as soon as I solve my problem with my ISP

**Comment by DavisBrown723** [comment]
Does this issue still garner any interest? Will close if not.
